### ISSUE TITLE
EducAndOS en Portátil HP Probook 455 g7

### ISSUE THREAD

**Issue by MMONTAVEZ** [aosucas499/guadalinex#7]
Muy buenas. He instalado desde la ISO que colgaste en un Portátil HP Probook 455 g7 y no he obtenido los driver para Promethean. Y en el menú de Clonedzilla, no me aparecia como primera opción la que tú apuntaba. Así que lo instalé desde la opción "Others" y siguiendo tus instrucciones no me aparecía el icono de Promethean en el escritorio. La conexión a la pizarra es por HDMI.
Disculpa la "chapa" en domingo. Ya sabes cómo de intensos son los compis de trabajo cuando algo no funciona.
Un saludo y gracias

**Comment by aosucas499** [comment]
Buenas tardes.

Imagino que habrás seguido estas instrucciones:

https://github.com/aosucas499/guadalinex/wiki/Promethean

Acabo se ver que hay un fallo en la imagen que te dice cuál es la opción a usar. En la imagen viene que uses la opción "instalar docker". Esa opción no es, selecciona la opción "instalar drivers promethean" y me cuentas. 

Un saludo.

**Comment by MMONTAVEZ** [comment]
No seguí esas instrucciones. Primero instalé desde la imagen para Portátil HP Probook 455 g7
https://github.com/aosucas499/guadalinex/wiki/hp y ahí es cuando no me aparece la imagen de Clonedzilla que dices: "marcar la primera opción"
La otra imagen no la probé porque entendí que no era para los portátiles hp

**Comment by aosucas499** [comment]
Estás seguro que tu hp es el 455? Te lo digo porque el mío lo es y hace dos días lo instalé. Además hay dos modelos hp de los que dieron en los coles. Si sigues estando seguro de que tu hp es el 455, vuelve a descargar la imagen y usa otro pendrive porque ya te digo que está probada hace dos días.

**Comment by MMONTAVEZ** [comment]
Sí, es el hp mencionado, lo tengo aquí en casa. Voy a probar aquí la imagen
que descargué y te comento.

**Comment by MMONTAVEZ** [comment]
Este es el menú de opciones que me aparece al arrancar desde la imagen para
el 455 g7

**Comment by aosucas499** [comment]
No se ve la imagen.

**Comment by MMONTAVEZ** [comment]
Ahora se ve?

**Comment by aosucas499** [comment]
No. Veo que estás escribiendome por email. Entra ahí en el enlace #7 y contesta subiendo la foto en el mensaje.

https://github.com/aosucas499/guadalinex/issues/7

**Comment by MMONTAVEZ** [comment]
A ver si se ve ahora
![IMG_20210919_192542](https://user-images.githubusercontent.com/83212471/133938073-4821f0f3-39e2-4237-86a4-bf943e06ca02.jpg)

**Comment by MMONTAVEZ** [comment]
Ahora sí?

**Comment by aosucas499** [comment]
Sí. Te faltan un par de resoluciones, pero si te carga el sistema con la primera de tus opciones no debería tener problemas. Sigue el proceso de instalación y manda aquí fotos del fallo. Esta semana en cuánto tengo un rato te contesto.

**Comment by MMONTAVEZ** [comment]
Muchas gracias, sé que todo esto no te lo pagan!!!

**Comment by MMONTAVEZ** [comment]
Muy buenas, A ver, descargué de nuevo la imagen e intenté hacer el usb de arranque desde la aplicación etcher. Pero me exigía una contraseña de superusuario y de ahí no pude pasar. 
Las anteriores imágenes las hice desde la aplicación etcher pero descargada para Windows10. ¿Está ahí el fallo? ¿Cómo utilizo la aplicación Etcher desde los dispositivos Linux?
Gracias compañero.

**Comment by aosucas499** [comment]
Si te pide una contraseña de superusuario y no la tienes es porque lo haces de educaandos, verdad? Si es desde educandos oficial no podrás usar etcher porque no tienes la contraseña root. Haz el pendrive con etcher desde un sistema con Windows, Ubuntu o educaandos plus.

**Comment by MMONTAVEZ** [comment]
Ok, volveré a intentarlo, aunque creo que hice el disco de arranque desde
windows.
Gracias

**Comment by aosucas499** [comment]
Los errores más comunes son algún fallo en la descarga o usar otro pendrive. Algunos pendrives parece que van bien pero fallan al grabar una iso. Prueba otro pendrive si te falla.

**Comment by aosucas499** [comment]
Me alegro.
